src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -103,21 +103,6 @@ class G1RemSetScanState : public CHeapObj<mtGC> {
   // to (>=) HeapRegion::CardsPerRegion (completely scanned).
   uint volatile* _card_table_scan_state;
 
-  // Return "optimal" number of chunks per region we want to use for claiming areas
-  // within a region to claim. Dependent on the region size as proxy for the heap
-  // size, we limit the total number of chunks to limit memory usage and maintenance
-  // effort of that table vs. granularity of distributing scanning work.
-  // Testing showed that 64 for 1M/2M region, 128 for 4M/8M regions, 256 for 16/32M regions,
-  // and so on seems to be such a good trade-off.
-  static uint get_chunks_per_region(uint log_region_size) {
```

Comment:
I actually think the concrete examples (`64 for 1M/2M region ...`) make this method easy to comprehend. Could you keep them?